### DISCUSSION TITLE
Coming from MUI5 Grid....

### DISCUSSION THREAD

**Discussion by MDrooker** [mantinedev/mantine#1336] [Q&A] [answered]
Hey-
I coming from the MUI5 framework...and had a question-
Is there a good place to see how to design UI's with Mantine, and more specifically the grid?

Coming from Flexbox, the MUI grid had translated that well.  
Am I overcomplicating Mantine's Grid.Col/Grid components?

**Comment by rtivital** [comment, marked as answer]
I haven't ever use mui, so cannot advise anything, take a look at Grid, Group, Stack and SimpleGrid components, they should cover all essential layouts, also see other examples – https://ui.mantine.dev/category/grids
